rest/data/host.go

```diff
@@ -69,6 +69,15 @@ func (hc *DBHostConnector) FindHostById(id string) (*host.Host, error) {
 	return h, nil
 }
 
+func (hc *DBHostConnector) FindHostByIP(ip string) (*host.Host, error) {
```

Comment:
Nit: Can you please add a comment above this function, similar to FindHostById?

Reply:
done